test/HistoryTest.cs

```diff
@@ -118,136 +119,289 @@ public void SensitiveHistoryOptionalBehavior()
 
             var options = PSConsoleReadLine.GetOptions();
             var oldHistoryFilePath = options.HistorySavePath;
-            var oldHistorySaveStyle = options.HistorySaveStyle;
-
+            var oldHistorySaveStyle = options.HistorySaveStyle;
```

Comment:
Did the newlines change?

Reply:
Yes, changed from `CRLF` to `LF`.

Reply:
https://github.com/PowerShell/PSReadLine/pull/1068/files?w=1 could be helpful